Cargo.toml

```diff
@@ -39,6 +39,7 @@ either = { version = "1.9", optional = true }
 eyre = { version = ">= 0.6.8, < 0.7", optional = true }
 hashbrown = { version = ">= 0.14.5, < 0.16", optional = true }
 indexmap = { version = ">= 2.5.0, < 3", optional = true }
+jiff = { version = "0.1.17", optional = true }
```

Comment:
I would suggest ensuring that the feature is named `jiff-01`, either by introducing a new feature or renaming the package. For example, [this is what `rust-postgres` does](https://github.com/sfackler/rust-postgres/blob/d7ccc7e398755addf13096b55bdab49c2cc96156/postgres-types/Cargo.toml#L50).

This ensures there is a paved path to supporting multiple versions of Jiff when a semver incompatible release comes out without making a semver incompatible release of `pyo3`.

Arguably, this ought to be done for other dependencies, but maybe they don't do incompatible releases often. For Jiff, I'd like get 1.0 out this summer, and then it should sit there for a long while.

Reply:
That's a good idea (especially if we know there will be a major version bump soon-ish), thanks for bringing that up. Renaming the package looks like a good option to me.